src/app/access-requests/features/access-request-manager-dialog/access-request-manager-dialog.component.html

```diff
@@ -6,6 +6,23 @@ <h2 mat-dialog-title>Access Request Detail</h2>
         <th class="pe-1 text-start">Dataset ID:</th>
         <td>{{ data.dataset_id }}</td>
       </tr>
+      @if (data.ticket_id) {
+        <tr>
+          <th class="pe-1 text-start">Relevant Issue:</th>
+          <td>
+            <a [href]="ticketUrl()" target="_blank"
```

Comment:
The anchor steals focus when first opening the dialog. We should maybe change the tabindex or something here.